app/views/ctc/ctc_pages/home.html.erb

```diff
@@ -10,7 +10,7 @@
     <div class="grid-flex contained column-row space-between">
       <div class="grid-flex column-column spacing-below-35">
         <h1 class="landing-page-header">
-          <%= t("views.ctc_pages.home.header") %>
```

Comment:
page title and the main h1 were different, which does not seem to have been on purpose.